packages/instantsearch.js/src/__tests__/common-connectors.test.tsx

```diff
@@ -109,13 +109,22 @@ const testSetups: TestSetupsMap<TestSuites> = {
         <button data-testid="RefinementList-refine">
           refine
         </button>
+      <button data-testid="RefinementList-refine-value">
```

Comment:
not sure if this is the best way to test it, maybe the value we're refining / making a createURL is configurable when you click on it (through an input for example)